libmscore/measurenumber.cpp

```diff
@@ -57,6 +57,9 @@ QVariant MeasureNumber::propertyDefault(Pid id) const
 
 void MeasureNumber::layout()
       {
+      if (!measure())
+            return; // Fix #301847.
```

Comment:
I don't think that comment is needed or even feasible, we don't do this usually, `git blame` would tell anyhow.

Reply:
I saw it somewhere but indeed, git blame will tell anyhow (as long we have a git blame :-))

Reply:
That's why I wrote "usually" ;-)